Revise ZHA config section order (#42851)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

## Configuration requirements

{% important %}
Be sure to connect a compatible radio module and restart Home Assistant before proceeding with configuration.
{% endimportant %}

ZHA will automatically create a Zigbee network once configured with a Zigbee coordinator; you can then add compatible devices.

It is strongly encouraged to review the guidance for [Zigbee interference avoidance and network range/coverage optimization)](#zigbee-interference-avoidance-and-network-rangecoverage-optimization).

{% include integrations/config_flow.md %}

1. In the popup dialog, select the **Serial Device Path** from the detected options on your system.
    - Choose the one to which your radio is connected.
2. Select **Submit**.
3. After submitting, the {% term integration %} will try to detect the radio type automatically.
4. If unsuccessful, you will need to manually set your radio type: 
    - Choose your **Radio Type**:
        - **ezsp**: Silicon Labs EmberZNet protocol (e.g., Home Assistant ZBT-1 or ZBT-2, Elelabs, HUSBZB-1, Telegesis)

>         - **deconz**: dresden elektronik deCONZ protocol (e.g., ConBee I/II, RaspBee I/II)

Reviewer: ```suggestion
        - **deCONZ**: dresden elektronik deCONZ protocol (e.g., ConBee I/II, RaspBee I/II)
```

>         - **znp**: Texas Instruments (e.g., CC253x, CC26x2, CC13x2)

Reviewer: ```suggestion
        - **znp**: Texas Instruments (for example, CC253x, CC26x2, CC13x2)
```

        - **zigate**: ZiGate Serial protocol (e.g., ZiGate USB-TTL, PiZiGate, ZiGate WiFi)

>         - **xbee**: Digi XBee ZB Coordinator Firmware protocol (e.g., Digi XBee Series 2, 2C, 3)

Reviewer: ```suggestion
        - **ezsp**: Silicon Labs EmberZNet protocol (for example, Home Assistant ZBT-1 or ZBT-2, Elelabs, HUSBZB-1, Telegesis)
        - **deCONZ**: dresden elektronik deCONZ protocol (for example, ConBee I/II, RaspBee I/II)
        - **znp**: Texas Instruments (for example, CC253x, CC26x2, CC13x2)
        - **zigate**: ZiGate Serial protocol (for example, ZiGate USB-TTL, PiZiGate, ZiGate WiFi)
        - **xbee**: Digi XBee ZB Coordinator Firmware protocol (for example, Digi XBee Series 2, 2C, 3)
```
we prefer "for example", or "such as" over e.g., see also https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

    - Select **Submit** to proceed to the next step.
5. Enter the **Serial device path**:
    - Most devices need at the very least the serial device path, such as `/dev/ttyUSB0`, but it is recommended to use device path from `/dev/serial/by-id` folder (e.g., `/dev/serial/by-id/usb-Silicon_Labs_HubZ_Smart_Home_Controller_C0F003D3-if01-port0`).
    - A list of available device paths can be found in {% my hardware title="Settings > System > Hardware" %} > **dot menu** > **All Hardware**.
6. Set the **Port speed** (not applicable for all radios)

> 7. Set the **Data flow control** (not applicable for all radios)

Reviewer: ```suggestion
6. Set the **Port speed** (not applicable for all radios).
7. Set the **Data flow control** (not applicable for all radios).
```
